### PR TITLE
add colophon and code of conduct

### PR DESCRIPTION
**Pull request by bluesomewhere**
also updated footer and sitetree. update to urls.py includes reordering entries, moved static pages to the top.

### CODE REVIEW

**Review by bluesomewhere — COMMENTED** [review]
note, commit message is incorrect. adds *colophon* under about, not coc. coc is already in nav.

File: pinaxcon/templates/static_pages/harassment-procedure-attendee.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+The staff is well informed on how to deal with the incident and how to further proceed with the situation.
+
+3\. If everyone is presently physically safe, involve law enforcement or security only at a victim's request. If you do feel your safety in jeopardy please do not hesitate to contact local law enforcement by dialing 911. If you do not have a cell phone, you can use any hotel phone or simply ask a staff member.
+
+**Note**: Incidents that violate the Code of Conduct are extremely damaging to the community, and they will not be tolerated. The silver lining is that, in many cases, these incidents present a chance for the offenders, and the community at large, to grow, learn, and become better. North Bay Python staff requests that they be your first resource when reporting a North Bay Python-related incident, so that they may enforce the Code of Conduct and take quick action toward a resolution.
+
+A listing of [North Bay Python staff is located here](/about/staff), including contact phone numbers. If at all possible, all reports should be made directly to [Ewa Jodlowska](mailto:ewa@python.org) (Event Coordinator) or [Jesse Noller](mailto:jnoller@python.org) (PyCon Chair).
```
Comment (chrisjrn) [inline pinaxcon/templates/static_pages/harassment-procedure-attendee.md]: Probably worth deleting these names.

File: pinaxcon/templates/static_pages/harassment-procedure-staff.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+Depending on the incident, the conference chair, or designate, may decide to make one or more public announcements. If necessary, this will be done with a short announcement either during the plenary and/or through other channels. No one other than the conference chair or someone delegated authority from the conference chair should make any announcements. No personal information about either party will be disclosed as part of this process.
+
+If some attendees were angered by the incident, it is best to apologize to them that the incident occurred to begin with.  If there are residual hard feelings, suggest to them to write an email to the conference chair or to the event coordinator. It will be dealt with accordingly.
+
+A listing of [North Bay Python staff is located here](/about/staff), including contact phone numbers. If at all possible, all reports should be made directly to [Ewa Jodlowska](mailto:ewa@python.org) (Event Coordinator) or [Jesse Noller](mailto:jnoller@python.org) (PyCon Chair).
```
Comment (chrisjrn) [inline pinaxcon/templates/static_pages/harassment-procedure-staff.md]: Names again.

**Review by chrisjrn — COMMENTED** [review]
This generally looks good, except that we appear to be publishing names of PyCon US staff.

**Comment by bluesomewhere** [comment]
OK, removed old contact info.

**Comment by chrisjrn** [comment]
This is generally OK, should replace "the Software Freedom" with "Software Freedom"

**Pull request merged**